navigation.php

```diff
@@ -24,6 +24,7 @@
             'Symfony' => 'docs/guides/symfony',
             'Pest From Scratch' => 'docs/guides/pest-from-scratch',
             'Creating Plugins' => 'docs/guides/plugins',
+            'Drift' => 'docs/guides/drift',
```

Comment:
Can we move the **Drift** guide up one line?

The **Creating Plugins** guide makes for a nice transition to the **Plugins** section :)

Reply:
Definitely makes sense. 👍🏻

Reply:
Thanks 👍 

Hope I'm not pushing it, but I had another idea 🙃

I like your idea of putting **Drift** on the guide section.
It would make even more sense if it was right after the **PHPUnit** guide.

Here is a transition that we could add:

> […] the test now contains only what really matters.
>
> Wouldn't it be cool if you could automatically migrate your PHPUnit tests to Pest? We got you! In the next section, you will learn all about Drift, our experimental migration tool 🔥
>
> Next section: Drift →

Reply:
Haha, that's a great idea. 👍🏻 Give me a few minutes and I'll try it out.

Reply:
Perfect!
I’ll merge it when I get back home 🙂